ios/engine/KMEI/KeymanEngine/Classes/KeymanPackage.swift

```diff
@@ -51,6 +51,24 @@ public class KeymanPackage {
   public var version: String? {
     return metadata.version
   }
+
+  var resources: [KMPResource] {
+    fatalError("abstract base method went uninplemented by derived class")
+  }
+
+  /**
+   * Returns an array of arrays corresponding to the available permutations of resource + language for each resource
+   * specified by the package.
+   *
+   * Example:  nrc.en.mtnt supports three languages.  A package containing only said lexical model would return an array
+   * with 1 entry:  an array with three InstallableLexicalModel entries, one for each supported language of the model, identical
+   * aside from language-specific metadata.
+   */
+  public var installableResourceSets: [[LanguageResource]] {
+    return resources.map { resource in
+      return resource.installableResources
+    }
+  }
```

Comment:
While not utilized yet, this provides the full set of metadata needed for a resource + language selection screen for installation prompts.  Each (leaf) member corresponds to one valid selectable pairing (keyboard/model + language id) available within the KMP.